src/android/target.rs

```diff
@@ -158,6 +158,14 @@ impl<'a> TargetTrait<'a> for Target<'a> {
         })
     }
 
+    fn name_list() -> &'static [&'a str]
+    where
+        Self: 'static,
+    {
+        static INSTANCE: OnceCell<Vec<&str>> = OnceCell::new();
+        INSTANCE.get_or_init(|| Self::all().keys().copied().collect::<Vec<_>>())
+    }
```

Comment:
Seems like we can avoid using `OnceCell` here by manually writing an array of the keys. 
```suggestion
    fn name_list() -> &'static [&'static str] {
        &["aarch64", "armv7", "i686", "x86_64"]
    }
```